pkg/server/sandbox_run.go

```diff
@@ -452,6 +452,27 @@ func (c *criService) generateSandboxContainerSpec(id string, config *runtime.Pod
 	}
 	g.SetProcessOOMScoreAdj(adj)
 
+	assetAnnotations := []string{
```

Comment:
maybe if the constants were already in an array of strings... specified in vc.go so you don't have to modify this block each time you add a new annotation.

Reply:
Since I will vendor it, I will need that array of strings to confine the selection of annotations from all the available ones which may be relevant for this task.